rds/sql/dialect/postgreSQL.ts

```diff
@@ -252,6 +257,11 @@ export abstract class AbstractPostgreSqlDialect extends diaImpl.ANSI {
         END;
       $BODY$;
       -- ALTER FUNCTION ${objWfDefnName}${argTypes} OWNER TO (TODO: OWNER);
+      INSERT INTO plpgsql_check_result
```

Comment:
@Kunjahamed-P @divyaSkumarCI01188 the `plpgsql_check_result` table name should not be hard-coded. Please use SQL interpolation to dynamically generate the proper name.

Reply:
@shah @Kunjahamed-P 
We will check it and update.